docs/cluster-management/dependencies.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: main
+title: Dependencies
+category: Cluster Management
+menu: menu
+toc:
+    - title: External Dependencies
+      url: "#external-dependencies"
+      active: true
+---
+## External Dependencies
+
+ | Technology             | Description | License |
+ | ----                   | ----        | ----    |
+ | Bitbucket/Github/Github Enterprise/Gitlab | Source Control Management System. Users must have access to one of the supported SCMs |         |
+ | Docker                 | Used with [executor-k8s](https://github.com/screwdriver-cd/executor-k8s) & [executor-docker](https://github.com/screwdriver-cd/executor-docker). Also required for running Screwdriver with [SD in a box](https://github.com/screwdriver-cd/in-a-box)  | [Apache 2.0](https://www.docker.com/legal/components-licenses) |
+ | Ember.js               | Screwdriver UI            | [MIT](https://github.com/emberjs/ember.js/blob/master/LICENSE)         |
+ | Golang                 | [Launcher](https://github.com/screwdriver-cd/launcher) & [log-service](https://github.com/screwdriver-cd/log-service) are written in Golang  | [License](https://golang.org/LICENSE) |
+ | Habitat                | For packaging dependencies like curl in the [Launcher](https://github.com/screwdriver-cd/launcher) container so that build containers have minimal required tooling | [Apache 2.0](https://www.habitat.sh/legal/licensing) |
+ | Hapi.js                | API & Store web servers are based on Hapi | [License](https://github.com/hapijs/hapijs.com/blob/master/LICENSE)        |
+ | Helm (Optional)           | For [deploying Screwdriver into k8s](https://github.com/screwdriver-cd/screwdriver-chart) | [Apache 2.0](https://github.com/helm/helm/blob/master/LICENSE) |
+ | HyperContainer (Optional) | Hypervisor-agnostic Docker runtime. Used with [executor-k8s-vm](https://github.com/screwdriver-cd/executor-k8s-vm) | [Apache 2.0](https://github.com/hyperhq/hyperd/blob/master/LICENSE) |
+ | Jenkins (Optional)     | Used in [executor-jenkins](https://github.com/screwdriver-cd/executor-jenkins)  | [MIT](https://jenkins.io/license) |
+ | JWT                    | For authentication & authorization | [MIT](https://github.com/jsonwebtoken/jsonwebtoken.github.io/blob/master/LICENSE.txt) |
+ | Kubernetes             | For running Screwdriver at scale using [executor-k8s](https://github.com/screwdriver-cd/executor-k8s) | [Apache 2.0](https://github.com/kubernetes/kubernetes/blob/master/LICENSE) |
+ | MariaDB(Optional) | One of the options for Database | [GPL Licence](https://mariadb.com/kb/en/library/licensing-faq/) |
+ | MySQL(Optional) | One of the options for Database | [GPL or Commercial](https://www.mysql.com/about/legal/) |
```

Comment:
wonder if these should all be linked to datastore-sequelize

Reply:
updated